.buildkite/pipeline.yml

```diff
@@ -1,12 +1,20 @@
+env:
+  SETUP_GVM_VERSION: 'v0.5.0' # https://github.com/andrewkroh/gvm/issues/44#issuecomment-1013231151
+  DOCKER_COMPOSE_VERSION: "1.25.5" # "v2.15.1"
```

Comment:
Using latest docker-compose version (v2.15.1) failed booting up the terraform scenarios (aws and gcp packages), with this error:

```
023/02/07 19:00:45 DEBUG running command: /opt/buildkite-agent/builds/bk-agent-elastic-gcp-1675796088139803659/elastic/elastic-package/bin/docker-compose -f /var/lib/buildkite-agent/.elastic-package/deployer/terraform/terraform-deployer.yml -f /opt/buildkite-agent/builds/bk-agent-elastic-gcp-1675796088139803659/elastic/elastic-package/test/packages/parallel/gcp/data_stream/compute/_dev/deploy/tf/env.yml -p elastic-package-service config
WARN[0000] The "TF_DIR" variable is not set. Defaulting to a blank string.
1 error(s) decoding:
 
* error decoding 'Volumes[0]': invalid spec: :/stage: empty section between colons
2023/02/07 19:00:45 DEBUG waiting for 1s before tearing down...
Error: error running package system tests: could not complete test run: could not setup service: could not get Docker Compose configuration for service: exit status 15
```

This error would be caused by the volume definition in this docker-compose file:
https://github.com/elastic/elastic-package/blob/main/internal/install/_static/terraform_deployer.yml#L15

Reply:
I think this is because `docker-compose config` didn't require to replace the environment variables in 1.x and it does in 2.x.

I have a fix for this in https://github.com/elastic/elastic-package/pull/1140/files#diff-37d9cb502e75dd46add8cb1fe8a9701aab162575bc068c9891b85ec55a305336L58

But feel free to bring it here if you want to use v2 (and close https://github.com/elastic/elastic-package/issues/1116 if you do :slightly_smiling_face: )

Reply:
I would prefer to keep this PR using the same docker-compose version as it runs in Jenkins, to ensure that there are no unexpected changes in behavior.

One note to remark about the tests run in docker-compose v2 is that is much more verbose and the buildkite output per step is truncated.